### ISSUE TITLE
Unique text property for timeline lines

### ISSUE THREAD

**Issue by alphapapa** [Fuco1/org-timeline#2]
This looks neat!  Would you mind adding a unique text property to each line you add to the buffer? That way, org-super-agenda could easily put the timeline wherever the user desires in the grouped agenda view.

**Comment by Fuco1** [comment]
Do you mean each line of the timeline has the same unique ID or every line really has a different unique one?  I guess the former, but just in case..

**Comment by alphapapa** [comment]
Yes, I meant that each line could have, e.g. the `org-agenda-timeline` property set to `t`.

**Comment by Fuco1** [comment]
@alphapapa 

So I've added an `org-timeline` to the timeline lines but it seems like they are not picked up by `org-super-agenda`.

I'm using this to define the section

```
(:name "Timeline"
 :pred (lambda (x) (get-text-property 0 'org-timeline x)))
```

**Comment by Fuco1** [comment]
Oh, this happens because you are using an advice on `org-agenda-finalize-entries`.

Do you think it would be too much trouble to use the finalize hook?  Otherwise I don't see how to make this work together :/
